draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -8509,6 +8475,20 @@ Content-Range: exampleunit 11.2-14.3/25
    A 200 response is heuristically cacheable; i.e., unless otherwise indicated by
    the method definition or explicit cache controls (see <xref target="heuristic.freshness"/>).
 </t>
+<t>
+   In 200 responses to GET or HEAD, an origin server &SHOULD; send any
+   available validator fields (<xref target="response.validator"/>) for the
+   <x:ref>selected representation</x:ref>, with both a strong entity-tag and
+   a <x:ref>Last-Modified</x:ref> date being preferred.
+</t>
+<t>
+   In 200 responses to state-changing methods, any validator fields
```

Comment:
```suggestion
   In 2xx responses to state-changing methods, any validator fields
```

Or is this really specific to 200?

Reply:
This is the section on 200. Yes, it is specific to 200. 201 has a slightly different requirement, and the other 2xx codes have unique requirements on metadata that don't need to be changed. The more generic statement (that validators in response to state-changing methods apply to the selected representation after the state change) is still in the section on validators.